chex/__init__.py

```diff
@@ -58,6 +60,7 @@
 from chex._src.asserts import assert_trees_all_equal
 from chex._src.asserts import assert_trees_all_equal_comparator
 from chex._src.asserts import assert_trees_all_equal_dtypes
+from chex._src.asserts import assert_trees_all_equal_sizes
```

Comment:
Missing this one in \_\_all\_\_.

Reply:
Good spot, this is sorted in the latest commit.